src/project/FileTreeView.js

```diff
@@ -518,6 +518,15 @@ define(function (require, exports, module) {
                 case "gif":
                     fileType = "image";
                     break;
+                case "eot":
+                case "woff":
+                case "otf":
+                case "ttf":
+                    fileType = "font";
+                    break;
+                case "mp3":
```

Comment:
Good catch, I just added the one filetype for testing and forgot to add the full list of supported types.